src/components/views/messages/MImageBody.js

```diff
@@ -496,7 +496,7 @@ export class HiddenImagePlaceholder extends React.PureComponent {
         return (
             <div className={className}>
                 <div className='mx_HiddenImagePlaceholder_button'>
```

Comment:
Can this become an AccessibleButton whilst you're here

Reply:
It's not a real button.